teaser/data/dataclass.py

```diff
@@ -41,23 +40,52 @@ class DataClass(object):
 
     def __init__(self, type_element_file=None):
         '''Constructor of DataClass
-
         '''
+
         if type_element_file is None:
             self.path_tb = utilitis.get_full_path(
                 "data/input/inputdata/TypeBuildingElements.xml")
         else:
             self.path_tb = utilitis.get_full_path("data/input/inputdata/"+str(
                 type_element_file))
 
-        __xml_file_tb = open(self.path_tb, 'r')
-        self.element_bind = tb_bind.CreateFromDocument(__xml_file_tb.read())
+        version_parse = element_tree.parse(self.path_tb)
+        __xml_file_tb = open(self.path_tb, 'r', encoding='utf-8')
+
+        if bool(version_parse.getroot().attrib) is False:
+            warnings.warn("You are using an old version of type building element data base XML file")
+            import teaser.data.bindings.v_0_3_9.typeelement_bind as tb_bind
+            self.element_bind = tb_bind.CreateFromDocument(__xml_file_tb.read())
+        elif version_parse.getroot().attrib['version'] == "0.3.9":
+            import teaser.data.bindings.v_0_3_9.typeelement_bind as tb_bind
+            self.element_bind = tb_bind.CreateFromDocument(__xml_file_tb.read())
```

Comment:
exactly, this is for other versions if you have a nicer way, go ahead!

I'm just preparing version 0.4:

´´´python
        elif version_parse.getroot().attrib['version'] == "0.3.9":
            import teaser.data.bindings.v_0_3_9.material_bind as mat_bind
            self.material_bind = mat_bind.CreateFromDocument(__xml_file_mat.read())
        elif version_parse.getroot().attrib['version'] == "0.4":
            import teaser.data.bindings.v_0_4.material_bind as mat_bind
            self.material_bind = mat_bind.CreateFromDocument(__xml_file_mat.read())
```